src/components/postHtmlRenderer/postHtmlRenderer.tsx

```diff
@@ -137,7 +137,32 @@ export const PostHtmlRenderer = memo(({
         const data = parseLinkData(tnode);
         _handleOnLinkPress(data);
       };
-
+      
+      if (tnode.classes?.indexOf('markdown-video-link') >= 0) {
+        // get video src 
+        let videoHref = tnode.attributes['data-embed-src'] || tnode.attributes['data-video-href'] || tnode.children[0].attributes['src'];
+        
+        return (
+          <WebView
+            scalesPageToFit={true}
+            bounces={false}
+            javaScriptEnabled={true}
+            automaticallyAdjustContentInsets={false}
+            onLoadEnd={() => {
+              console.log('load end');
+            }}
+            onLoadStart={() => {
+              console.log('load start');
+            }}
+            source={{ uri: videoHref }}
+            style={{ width: contentWidth, height: (contentWidth * 9) / 16 }}
+            startInLoadingState={true}
+            onShouldStartLoadWithRequest={() => true}
+            mediaPlaybackRequiresUserAction={true}
+            allowsInlineMediaPlayback={true}
+          />
+        );
+      }
```

Comment:
for youtube... the class is `markdown-video-link-youtube` and attributes are as follows

data-youtube: contains video id
data-start-time: contains start time of video (optional)

pass this data to youtube iframe implemented inside videoPlayerSheet.tsx and it should play without issues.